code/public/index.html

```diff
@@ -4,31 +4,19 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>Technigo React App</title>
+    <meta name="description" content="Music Radar survey" />
+    <meta name="author" content="Jamie Cook" />
+    <meta property="og:title" content="Music Radar survey" />
+    <meta property="og:description" content="Music Radar survey project for the technigo boot camp" />
+    <meta property="og:image" content="https://jamiecook-survey.netlify.app/assets/festival-image.jpg" />
+    <meta property="og:url" content="https://jamiecook-survey.netlify.app" />
```

Comment:
It's nice that you added og tags, Jamie. I tried sending the live website link on facebook and found out the image did not show. I guess it is because the "festival-image.jpg" is stored in assets in the src folder which will not be shown in the browser when we deploy the site on Netlify. You can move/copy the image to Public folder instead to make it work. At least, that's how I did it. I am sure there is another ways too. Check this out on our class Slack channel where Sara (our Sara, yes :-)) has the same problem. https://technigo.slack.com/archives/C6QAN25EV/p1603473342434300